core/tests/test_utils.py

```diff
@@ -73,11 +73,26 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Sets the content, status code and headers that the urlfetch mock
+        should return with.
+
+        Args:
+            content: str. The content to return in subsequent calls to the
+            urlfetch mock.
+            status_code: status_code for return_values.
```

Comment:
status_code: int. status_code for return_values. 

Mention the type. Ditto for the next line.